docs/zh-hant/docs/async.md

```diff
@@ -0,0 +1,442 @@
+# 並行與 async / await
+
+有關*路徑操作函式*的 `async def` 語法的細節與非同步 (asynchronous) 程式碼、並行 (concurrency) 與平行 (parallelism) 的一些背景知識。
+
+## 趕時間嗎?
+
+<abbr title="too long; didn't read（文長慎入）"><strong>TL;DR:</strong></abbr>
+
+如果你正在使用要求你以 `await` 語法呼叫的第三方函式庫，例如：
+
+```Python
+results = await some_library()
+```
+
+然後，使用 `async def` 宣告你的*路徑操作函式*：
+
+
+```Python hl_lines="2"
+@app.get('/')
+async def read_results():
+    results = await some_library()
+    return results
+```
+
+/// note | 注意
```

Comment:
I've fixed this (it wasn't rendering properly)
(we have to include the “note” as the styling depends on this part)